block/public.go

```diff
@@ -35,20 +35,19 @@ func NopMetrics() *Metrics {
 // DAClient is the interface representing the DA client for public use.
 type DAClient = da.Client
 
-// NewDAClient creates a new DA client with configuration
+// NewDAClient creates a new DA client backed by the blob JSON-RPC API.
 func NewDAClient(
-	daLayer coreda.DA,
+	blobRPC *blobrpc.Client,
 	config config.Config,
 	logger zerolog.Logger,
 ) DAClient {
 	return da.NewClient(da.Config{
-		DA:                       daLayer,
+		Client:                   blobRPC,
 		Logger:                   logger,
 		Namespace:                config.DA.GetNamespace(),
 		DefaultTimeout:           config.DA.RequestTimeout.Duration,
 		DataNamespace:            config.DA.GetDataNamespace(),
 		ForcedInclusionNamespace: config.DA.GetForcedInclusionNamespace(),
-		RetrieveBatchSize:        config.DA.RetrieveBatchSize,
```

Comment:
This should be reimplemented. This was a recent addition and this got lost in the refactor (https://github.com/evstack/ev-node/pull/2902)